homeassistant/components/mqtt/light/schema_basic.py

```diff
@@ -762,7 +693,8 @@ def scale_rgbx(color, brightness=None):
                     brightness = 255
                 else:
                     brightness = kwargs.get(
-                        ATTR_BRIGHTNESS, self._brightness if self._brightness else 255
+                        ATTR_BRIGHTNESS,
+                        self._attr_brightness if self._attr_brightness else 255,
```

Comment:
I suppose
```suggestion
                        self._attr_brightness or 255,
```
would be a more succinct way to spell this...

Reply:
My concern is that with attribute we should instead be checking the value:
```suggestion
                        self.brightness if self.brightness else 255
```
or
```suggestion
                        self.brightness or 255
```

`or` is more succinct, but that could be kept for a follow-up PR if we want to stay closer to the existing code.
Same applies to most of the other comments from @akx